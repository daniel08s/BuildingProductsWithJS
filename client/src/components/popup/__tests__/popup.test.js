/* global test expect shallow */
import Popup from '../index';

test('# Popup', () => {
  const wrapper = shallow(<Popup />);
  expect(wrapper).toMatchSnapshot();
});
