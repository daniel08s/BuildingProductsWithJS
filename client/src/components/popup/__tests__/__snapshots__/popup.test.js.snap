// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Popup 1`] = `
<div
  style={
    Object {
      "bottom": "25px",
      "maxWidth": "500px",
      "position": "absolute",
      "right": "17%",
      "width": "100%",
    }
  }
>
  <div
    className="row"
  >
    <div
      className="col-xs-11 col-sm-9 col-md-7"
    >
      <Connect(Notifications) />
    </div>
  </div>
</div>
`;
